### PR TITLE
Endeavor is not a "physical attack"

### PR DESCRIPTION
**Pull request by galbia**
Endeavor is more of a support move and shouldn't trigger a change in suggested nature.
Also alphabetized the section

### CODE REVIEW

File: js/client-teambuilder.js [hunk 1]
```diff
@@ -1535,7 +1535,7 @@
 						if (move.id === 'toxic' || move.id === 'leechseed' || move.id === 'willowisp') moveCount['Stall']++;
 						moveCount['Support']++;
 					}
-				} else if (move.id === 'rapidspin' || move.id === 'counter' || move.id === 'mirrorcoat' || move.id === 'metalburst') {
+				} else if (move.id === 'counter' || move.id === 'endeavor' || move.id === 'metalburst' || move.id === 'mirrorcoat' ||  move.id === 'rapidspin') {
```
Comment (RhythmsAreFromVibrations) [inline js/client-teambuilder.js]: Why did you put an extra space before rapid spin?

Comment (Zarel) [inline js/client-teambuilder.js]: ^

Comment (galbia) [inline js/client-teambuilder.js]: ugh that's a c/p error i didnt proofread sorry :/

**Comment by xfix** [comment]
Oops, I accidentally removed author info while fixing that for galbia.

File: js/client-teambuilder.js (hunk 1 above)
Comment (galbia) [inline js/client-teambuilder.js]: fixed

**Pull request merged**